src/content/text-detail/hi/about.mdx

```diff
@@ -1,27 +1,27 @@
 ---
-title: "About"
+शीर्षक: "विषय में "
```

Comment:
I think "परिचय" will be better for about.

Reply:
done